website/dynamic-blocks/props-table.js

```diff
@@ -8,58 +8,82 @@ const Th = ({ children }) => (
 	<th css={{ ...tableCellStyles, borderBottom: 'solid 3px #62194a' }}>{children}</th>
 );
 const Td = ({ children }) => <td css={{ ...tableCellStyles }}>{children}</td>;
+const Table = ({ data }) => {
+	console.log(data);
+	return (
+		<Fragment>
+			<h2>{data.name}</h2>
+			<table css={{ borderCollapse: 'collapse', width: '100%' }}>
+				<tbody>
+					<tr>
+						<Th>Property</Th>
+						<Th>Type</Th>
+						<Th>Required</Th>
+						<Th>Default</Th>
+					</tr>
+					{data.props.map((d, i) => (
```

Comment:
This will have to be a recursive call here... as we don't know how deep the shape might be.